chia/util/initial-config.yaml

```diff
@@ -195,6 +195,7 @@ harvester:
 
   # Plots are searched for in the following directories
   plot_directories: []
+  recursive_plot_scan: False # If True the harvester scans plots recursively in the provided directories.
```

Comment:
I wonder if this is better or if we should allow `/**` on the end of directories for recursion selection or something.

Reply:
And less flexible, yep.  Probably usually not an issue.